docs/CNAME

```diff
@@ -1 +0,0 @@
-docs.camel-ai.org
```

Comment:
Why deteleted?

Reply:
https://github.com/camel-ai/camel/blob/1b6856d7bdae5b766150bb9b0d661f0e95c3069e/.github/workflows/documentation.yml#L40

this setting has been included in doc workflow by @WHALEEYE , so this file is not needed